### DISCUSSION TITLE
Is there a dictionary on possible MQTT messages?

### DISCUSSION THREAD

**Discussion by ezequielandrush** [zwave-js/zwave-js#6522] [Q&A]
Hello all. I am setting a zwave-js gateway to mqtt. I could pair some devices and I also could operate them. The issue I'm having is I don't know how to send commands to the devices (for example, lock/unlock the door). I have found an example on the docs but I'm afraid I can't understand it very well.

Thanks in advance
Zeke

**Comment by AlCalzone** [comment]
There is, in the documentation for **Z-Wave JS UI**. I don't have the link ready, but you should find it by searching for mqtt there.
